requirements.txt

```
@@ -1 +1 @@
-discord.py ~= 1.2.5
\ No newline at end of file
+discord.py == 1.3.4
```

```suggestion
discord.py ~= 1.3.4
```
Or is there a good reason to be stricter on the version?